warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1284,35 +1284,35 @@ public boolean validateOptions(Map<String,String> options) {
             setTrackSizes(Boolean.parseBoolean(options.get(TRACK_SIZES)));
         }
 
-        if (options.containsKey(PROJECTION_FIELDS)) {
+        if (options.containsKey(INCLUDED_FIELDS)) {
             this.projectResults = true;
-            this.useWhiteListedFields = true;
-
-            String fieldList = options.get(PROJECTION_FIELDS);
-            if (fieldList != null && EVERYTHING.equals(fieldList)) {
-                this.whiteListedFields = UniversalSet.instance();
-            } else if (fieldList != null && !fieldList.trim().equals("")) {
-                this.whiteListedFields = new HashSet<>();
-                Collections.addAll(this.whiteListedFields, StringUtils.split(fieldList, Constants.PARAM_VALUE_SEP));
+            this.useIncludedFields = true;
+
+            String fieldList = options.get(INCLUDED_FIELDS);
+            if (EVERYTHING.equals(fieldList)) {
+                this.includedFields = UniversalSet.instance();
+            } else if (fieldList != null && !fieldList.trim().isEmpty()) {
+                this.includedFields = new HashSet<>();
+                Collections.addAll(this.includedFields, StringUtils.split(fieldList, Constants.PARAM_VALUE_SEP));
             }
             if (options.containsKey(HIT_LIST) && Boolean.parseBoolean(options.get(HIT_LIST))) {
-                this.whiteListedFields.add(JexlEvaluation.HIT_TERM_FIELD);
+                this.includedFields.add(JexlEvaluation.HIT_TERM_FIELD);
             }
         }
 
-        if (options.containsKey(BLACKLISTED_FIELDS)) {
+        if (options.containsKey(EXCLUDED_FIELDS)) {
             if (this.projectResults) {
-                log.error("QueryOptions.PROJECTION_FIELDS and QueryOptions.BLACKLISTED_FIELDS are mutually exclusive");
+                log.error("QueryOptions.INCLUDED_FIELDS and QueryOptions.EXCLUDED_FIELDS are mutually exclusive");
                 return false;
             }
 
             this.projectResults = true;
-            this.useBlackListedFields = true;
+            this.useExcludedFields = true;
 
-            String fieldList = options.get(BLACKLISTED_FIELDS);
-            if (fieldList != null && !fieldList.trim().equals("")) {
-                this.blackListedFields = new HashSet<>();
-                Collections.addAll(this.blackListedFields, StringUtils.split(fieldList, Constants.PARAM_VALUE_SEP));
+            String fieldList = options.get(EXCLUDED_FIELDS);
+            if (fieldList != null && !fieldList.trim().isEmpty()) {
```

Comment:
Slight java 11 improvement would be 
```suggestion
            if (fieldList != null && !fieldList.isBlank()) {
```

Reply:
done